hard/pickle/bsg_mem/bsg_mem_1rw_sync_mask_write_bit_macros.vh

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
are these macros used in more than one place in the code? Advise against factoring them out, if so.  this is going to be another thing on the include path.

the name space issues are a little concerning here as well.

Reply:
Yeah, the headers are used by tapeouts specifically. Separate headers is our methodology for hard mems (see gf14). The advantage is that you can replace the hard implementation (specific ram swaps) without modifying basejump, but including the macros to avoid duplicate code. This avoids the problem of needing to PR for every ram chance on a tapeout. 

If someone does a multi-node design using basejump, I’m sure they can figure out the namespace issue :)